scripts/mysql_slave_lag.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+seconds=`mysql -s -e 'show slave status\G' | grep Seconds_Behind_Master | awk {'print $2'}`
+test "$seconds" = "0"
```

Comment:
It seems like seconds behind master should be acceptable at some low number above 0.  If a slave is 3s behind the master, is that a problem?  Probably not.  45s?  Maybe.  300s?  Surely.